pkg/vulnsrc/nvd/nvd.go

```diff
@@ -26,21 +26,30 @@ const (
 	nvdSource   = "nvd@nist.gov"
 )
 
+type DB interface {
+	db.Operation
+	Put(*bolt.Tx, Cve) error
+}
+
 type VulnSrc struct {
-	dbc db.Operation
+	DB
+}
+
+type Nvd struct {
```

Comment:
nit: https://go.dev/wiki/CodeReviewComments#initialisms
```suggestion
type NVD struct {
```